app/app/analytics/private-cloud/ActiveProducts.tsx

```diff
@@ -11,7 +12,9 @@ export default function ActiveProducts({ data }: { data: ActiveProduct[] }) {
     <LineGraph
       index="date"
       title="Active Products"
-      subtitle="This graph shows the cumulative total of products provisioned through the registry"
+      subtitle={`This chart displays the cumulative number of products created from ${formatDate(
+        pageSnapshot.dates?.[0] || data?.[0]?.date,
+      )} to ${formatDate(pageSnapshot.dates?.[1] || new Date())}.`}
```

Comment:
please revisit the bunch of falsy checks; it looks complicated; try simplifying it.